pkg/operation/botanist/component/resourcemanager/resource_manager.go

```diff
@@ -882,15 +891,53 @@ func (r *resourceManager) ensureShootResources(ctx context.Context) error {
 				Namespace: metav1.NamespaceSystem,
 			}},
 		}
+
+		failurePolicy = admissionregistrationv1.Ignore
+		matchPolicy   = admissionregistrationv1.Exact
+		sideEffect    = admissionregistrationv1.SideEffectClassNone
+
+		podSchedulerNameMutatingWebhookConfiguration = &admissionregistrationv1.MutatingWebhookConfiguration{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:   "pod-scheduler-name",
+				Labels: appLabel(),
+			},
+			Webhooks: []admissionregistrationv1.MutatingWebhook{
+				{
+					Name: "pod-scheduler-name.scheduling.gardener.cloud",
+					Rules: []admissionregistrationv1.RuleWithOperations{{
+						Rule: admissionregistrationv1.Rule{
+							APIGroups:   []string{corev1.GroupName},
+							APIVersions: []string{corev1.SchemeGroupVersion.Version},
+							Resources:   []string{"pods"},
+						},
+						Operations: []admissionregistrationv1.OperationType{admissionregistrationv1.Create},
+					}},
+					NamespaceSelector:       &metav1.LabelSelector{},
+					ObjectSelector:          &metav1.LabelSelector{},
```

Comment:
Is this needed?  Why do you specify this explicitly?

Reply:
Maybe to make it clear that all namespaces and objects get selected/mutated. But if you say that we rather prefer to rely on the default values, then I can remove them.

Reply:
OK, I was just wondering why this is explicitly specified. If there is no reason for it then I vote for removing it, but I don't have a strong opinion.

Reply:
Hm, I think it wouldn't harm to keep them. It can also prevent wondering what is the default value when they are not specified.